apps/web-portal-client/lib/components/dialog/Dialog.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {IconButton} from './IconButton'
+import {ExitIcon} from './ExitIcon'
+
+export function Dialog({open, onClose, children}) {
+  if (!open) {
+    return <></>
+  }
+  return (
+    <div className="fixed inset-0 z-50 overflow-auto bg-smoke-light flex">
+      <div className="relative p-8 bg-white w-full max-w-md m-auto flex-col flex rounded-lg">
+        <div>{children}</div>
+        <span className="absolute top-0 right-0 p-4">
+          <IconButton onClick={() => onClose()}>
```

Comment:
why we need this "x" button?
you can add a closing dialog by clicking outside the dialog and make this optional (on/off by props passing)